docs/counter.md

```diff
@@ -82,22 +82,27 @@ The JA and KA inputs of FF-A are tied to logic 1. So FF-A will work as a toggle
 * **4 bit synchronous up counter:** In the up counter the 4 bit binary sequence starts from 0000 and increments up to 1111, i.e from 0 to 15.
 
 <div style="text-align:center"><img src="../assets/images/4bit_up_counter.png" height="300"/></div>
+
+<iframe width="400%" height="400px" src="https://circuitverse.org/simulator/embed/93827" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen></iframe>
```

Comment:
![image](https://user-images.githubusercontent.com/41790090/82753462-1d5de680-9dc6-11ea-937a-5f567ad273f2.png)
Please fix the width issue.
i think when you tried to change the 400px to 100% you made it 400%. 😄

Reply:
Fixed it to 100% @mahmodHammad